cmd/Makefile

```diff
@@ -0,0 +1,36 @@
+SUBDIRS := driver-kube-registrar \
+	driver-node-registrar
```

Comment:
minor nit. would calling it cluster-registrar make more sense then kubernetes-registrar? Its all kubernetes.

Reply:
Good question. I'll defer to others.